<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>magicItemAdder_window</class>
 <widget class="QDialog" name="magicItemAdder_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(160, 160, 160)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="weaponProperties_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
background-color:rgb(217,217,217);
padding:1px;
}</string>
     </property>
     <property name="text">
      <string>Please name your sheet:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" alignment="Qt::AlignTop">
    <widget class="QFrame" name="horizontalFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cancelSheetName_button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
background-color:rgb(255, 221, 221);
padding:4px;
}

QPushButton::hover{
background-color: rgb(255, 184, 184);
}

QPushButton::pressed{
background-color: rgb(255, 140, 140);
}

</string>
        </property>
        <property name="text">
         <string>Cancel save</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setSheetName_button">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
background-color:rgb(218, 255, 212);
padding:4px;
}

QPushButton::hover{
background-color: rgb(182, 255, 181);
}

QPushButton::pressed{
background-color: rgb(135, 255, 129);
}

</string>
        </property>
        <property name="text">
         <string>Set name and save</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignTop">
    <widget class="QFrame" name="horizontalFrame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: black;
background-color:rgb(217,217,217);
padding:3px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Sheet Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sheetName_entry">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>SizeVerCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color: white;
border-style: outset;
border-width: 1px;
border-radius: 6px;
border-color: black
}

QLineEdit::hover{
background-color: rgb(225, 225, 225);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
